dashboard-widgets.el

```diff
@@ -876,9 +886,9 @@ Argument IMAGE-PATH path to the image."
 ;;; Initialize info
 (defun dashboard-insert-init-info ()
   "Insert init info."
-  (let ((init-info (if (functionp dashboard-init-info)
-                       (funcall dashboard-init-info)
-                     dashboard-init-info)))
+  (let* ((init-time (dashboard-init--time))
+         (packages-count (dashboard-init--packages-count))
+         (init-info (funcall dashboard-init-info init-time packages-count)))
```

Comment:
I think the variable `dashboard-init-info` accepts a string as well. 🤔 See https://github.com/emacs-dashboard/emacs-dashboard/blob/master/docs/FAQ.org#how-can-i-change-my-init-message.